src/chocolatey.resources/helpers/functions/Install-ChocolateyPowershellCommand.ps1

```diff
@@ -51,7 +51,7 @@ it in the package next to the install script, the path will be like
 This is the 32 bit url to download the resource from. This resource can
 be used on 64 bit systems when a package has both a Url and Url64bit
 specified if a user passes `--forceX86`. If there is only a 64 bit url
-available, please remove do not use the paramter (only use Url64bit).
+available, please remove do not use the parameter (only use Url64bit).

```

Comment:
+1 - you don't need a ticket for a typo fix, especially if it is in docs. Just send a PR with `(doc) Fixed typo/spelling`. HTH.